sdk/service/service.go

```diff
@@ -85,5 +85,8 @@ func (s *Service) Create(srv *service.Service) error {
 		srv.Sid = "_" + srv.Hash
 	}
 
+	// apply main service key to service.
+	srv.Configuration.Key = service.MainServiceKey
```

Comment:
Why doing this here? It should be in done `server/grpc/api/service.go` in function `fromProtoConfiguration`.

Reply:
done